src/client.ts

```diff
@@ -276,4 +280,12 @@ export class KolibriClient {
     async nodeDeleteHistory(params: NodeDeleteHistoryParams): Promise<number> {
         return this.delegate.nodeDeleteHistory(params);
     }
+    
+    async nodeSubscribe(params: NodeSubscribeParams): Promise<number> {
```

Comment:
I prefer number only, otherwise we need to get and return all information for a node (properties + permissions for user, has subnodes, ...) and also for its sub-nodes in case of recursive flag. The amount of information to transmit would be to much and can be requested by the user if needed. Would only subscribe to events happening on this one and extend its functionality in case it is explicitly requests to be similar to userSubscribe RPC